fades/main.py

```diff
@@ -173,6 +175,10 @@ def go(argv):
     l.debug("Dependencies from parameters: %s", manual_deps)
     indicated_deps = _merge_deps(ipython_dep, indicated_deps, reqfile_deps, manual_deps)
 
+    # Check for packages updates
```

Comment:
Mmmm... we talked about once about checking in other thread, but that was when we talked to check new updates while doing normal work; here the user is requesting fades to verify, I think it's ok to wait...